client/src/Views/ServiceProviderViews/WalletPage.jsx

```diff
@@ -46,7 +47,7 @@ class WalletPage extends Component {
           {transactions.map((transaction, idx) => 
           <div key={idx}>
```

Comment:
Could you add a `className` to the container div called `transaction-container` for styling. Thanks